### ISSUE TITLE
How to reproduce maximum results

### ISSUE THREAD

**Issue by potato2734** [GraphPKU/Open-World-KG#1]
Hello, thanks for the great work. It really inspired my perspective of evaluation.

I am currently having trouble of reproducing your results of the ` full_test` vs `sparse_test_set` figure. In your figures, some models (RotatE : model 3 and 4) seem to reach 1.0 performance on MRR, Hits@k etc. I wanted to produce the results of my own so I follwed all the hyper parameters given in the appendix and tuned those that were missing (learning rate (lr) and adversarial temperature (a)).

I altered `test-full-hard-answers.pkl` into `test.txt` and `test-sparse-hard-answers.pkl` into `valid.txt` and trained on `train.txt` while evaluating on `valid.txt` and `test.txt` without early stopping. I also restricted the scope of filtered triples in validation to `train+valid` while allowing test phase to access all triples `train+valid+test`. However, I couldn't get near 1.0 performance and came short on max MRR of 0.74 for test set (trained on d=95, tie breaking protocol applied). 

Although test MRR of 0.74 and validation MRR of 0.57 is enough to show that MRR is inconsistent and produces degradation, the figure will be much prettier and consistent if I could achieve 1.0 on ` full_test`. Can you give me some tips on how to solve this problem? :)
